package.json

```diff
@@ -12,9 +12,10 @@
     "build": "./build.js",
     "migrate": "node migrate",
     "branchdb": "ts-node scripts/branchDb.ts",
+    "start": "NODE_OPTIONS=--max_old_space_size=8192 ./build.js --no-clear -run -watch --settings settings.json",
```

Comment:
This seems like it's reverting some changes that have already been merged into master... wonder if this is an artifact of whatever caused the issue where we had a bunch of merge commits in lw-deploy that needed to get merged back into master despite not changing any files